appointment/upload-documents/uploadDocumentsDataService.js

```diff
@@ -69,6 +69,39 @@ define('uploadDocumentsDataService', [], function($http) {
 				}, (error) => {
 					return null;
 				});
+			},
+			storeConsent: function(reviewConsent, virtualConsent, signature) {
```

Comment:
You should really avoid the `function` keyword and use the arrow functions like we do elsewhere in these files--mostly just for consistency, it's tomato-tomato functionally, but the arrow function is the more modern syntax